modules/node/handler/cms_node_pagination_builder.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Paginator builder for CMS nodes."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_NODE_PAGINATION_BUILDER
+
+inherit
+
+	CMS_PAGINATION_BUILDER [CMS_NODE]
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make (a_api: CMS_NODE_API)
+			-- Create an object.
+		do
+			node_api := a_api
+			count := 5
+			offset := 0
+		ensure
+			node_api_set: node_api = a_api
+			limit_set: count = 5
+			offset_set: offset = 0
+		end
+
+	node_api: CMS_NODE_API
+			-- CMS API.
+
+feature -- Pager
+
+	items: ITERABLE [CMS_NODE]
+			-- <Precursor>.
+		do
+				--NOTE: the current implementation does not use
```

Comment:
something like that
but maybe with a lighter CMS_PAGINATION ... that would have only offset, count, order, ... attributes.
and no  "items"

And instead of LIST, we could have  CMS_PAGINATED_LIST [CMS_NODE] ..
that would remind the offset, count, ... and so on ...
but still be conforming to ITERABLE [CMS_NODE]